content/pages/blogposts/2024-06-17-hybrid-search.md

```diff
@@ -0,0 +1,9 @@
+Title: Hybrid Search with Apache Solr
+category: solr/blogposts
+slug: hybrid-search-with-apache-solr
+URL: blogposts/hybrid-search-with-apache-solr.html
+save_as: blogposts/hybrid-search-with-apache-solr.html
+
+### This blog shows how to run a hybrid search(keyword-based search + vectors) in Apache Solr with code examples and explanations!
```

Comment:
The other blog posts use a h1 header, e.g.
```
# Hybrid Search with Apache Solr
```

Then the long-ish text "This blog..." could be plaintext.

Reply:
+1 to Jan's formatting tweaks

[0] Less crucially, I think this blurb might advertise the external post a bit better if it had a few more details: who wrote the external post, where is it hosted, a bit more on the "main takeaway", etc.  Maybe consider something like:

> Contrary to popular misconception, Solr already supports hybrid search (the combination of keywork and vector based search techniques)!  Hybrid search can be used in retrieval, ranking, and even Solr's commonly-used "Learning to Rank" module.  Alessandro Benedetti of Sease Ltd. explains more in a recent post, complete with code examples and explanation.
>
> Learn how you can start using hybrid search in your deployments today - the complete post can be read [here](https://sease.io/2023/12/hybrid-search-with-apache-solr.html)

Reply:
Great point. Perhaps we could end up with having a template `.md` file in the repo that folks can start from when they want to make a post that links to external blog.